CONTRIBUTING.md

```diff
@@ -12,7 +12,17 @@ Working on your first open source contribution? We are so excited to have you! Y
 #### How to submit
 We need all contributors to sign our [volunteer agreement](https://github.com/SexualHealthInnovations/callisto-core/blob/master/shi-volunteer-agreement.pdf) (PDF) in order to accept contributions. Signed agreements can be submitted to tech@sexualhealthinnovations.org. We can take fixes to documentation without a signed agreement.
 
-Coming soon: how to run tests
+#### How to run tests
+````
+git clone git@github.com:SexualHealthInnovations/callisto-core.git # first time only
+cd callisto-core
+pip install virtualenv # first time only
+virtualenv venv # localize your dependencies
+source venv/bin/activate
+pip install -r requirements-test.txt # requires postgres to be installed (brew install postgres)
```

Comment:
Oh, dang, I made a change so we wouldn't need postgres in this commit (https://github.com/SexualHealthInnovations/callisto-core/commit/c6481c5d57efca0eb203c5a1c9d1c02b01d2be45) but I forgot to remove the dependency on psycopg2 in requirements.txt. Could you remove that requirement for me and then update these instructions?